dockerfiles/init/modules/keycloak/files/che/login/messages/messages_en.properties

```diff
@@ -0,0 +1,240 @@
+product=Eclipse Che
```

Comment:
It will be needed for each update of the keycloak component. so the need will come sooner than never.